encfs/encfssh

```diff
@@ -6,16 +6,17 @@
 # Contributed by David Rosenstrauch.
```

Comment:
In addition to the arguments made in the linked articles (which I strongly suggest reading), explicit error handling is very much the Right Thing for this specific program and use case inasmuch as there's a large critical section (after the mount) following which any exit *without* an unmount is dangerously wrong.